library/src/main/java/org/xmtp/android/library/messages/DecryptedMessage.kt

```diff
@@ -8,5 +8,6 @@ data class DecryptedMessage(
     var encodedContent: EncodedContent,
     var senderAddress: String,
     var sentAt: Date,
-    var topic: String = ""
+    var topic: String = "",
+    var shouldPush: Boolean? = null,
```

Comment:
why wouldn't this always be a boolean with a default value of `false`?

Reply:
we are checking it here https://github.com/xmtp/xmtp-android/pull/164/commits/887f1f4dce979c57cf4d86d2fd035273fd70d78d#diff-5beaf5d0240fb991d7984f4f1dba63dcbafa3138e84bd5d510a688ad83446806R164:~:text=val-,calculatedShouldPush,-%3D%20shouldPush%20%3F